openapi/components/schemas/Plans/PlanTypes/OneTimeSalePlan.yaml

```
@@ -1,4 +1,101 @@
 type: object
 additionalProperties: false
-allOf:
-  - $ref: ../../Common/CommonPlan.yaml
+required:
+  - name
+  - currency
+  - productId
+  - pricing
+properties:
+  id:
+    type: string
+    description: ID of the plan.
+    maxLength: 50
+    example: plan_0YV7DENSVGDBW9S71XZNNYYQ0X
+  name:
+    description: |-
+      Name of the plan.
+      This name is displayed on invoices and receipts.
+    type: string
+    maxLength: 255
+  description:
+    type: string
+    description: |-
+      Plain-text description of the plan.
+      This field accepts plain-text only.
+  richDescription:
+    type: string
+    description: |-
+      Rich-text description of the plan.
+      This field accepts rich text formatting,
+      such as: bold, underline, italic, and hyperlinks.
```

I think this should be a one liner:

```suggestion
      This field accepts rich text formatting, such as: bold, underline, italic, and hyperlinks.
```